setup.sh

```diff
@@ -130,36 +136,45 @@ elif [[ $MODE == "nightly" ]]; then
     pip3 install tbp-nightly --upgrade
 elif [[ $MODE == "head" ]]; then 
 # Head mode
+    echo "Installing jax and jaxlib from head"
+    if [ -d "$HOME/jax" ]; then
+        cd "$HOME/jax" && git pull origin main
+    else
```

Comment:
This part of the CR is not mergeable -- it is duplicative but different from the nightly case.

Reply:
I researched this more and we need to do this differently from nightly because jaxlib lives in JAX and I need to link it to local openxla and pass in the `--enable-tpu` flag as well (https://jax.readthedocs.io/en/latest/developer.html#building-jaxlib-from-source-with-a-modified-xla-repository - the command in this link is being run from the jax directory)

Reply:
ok sounds good.